command/src/config.rs

```diff
@@ -1144,8 +1144,14 @@ fn default_accept_queue_timeout() -> u32 {
 }
 
 impl Config {
-    pub fn load_from_path(path: &str) -> io::Result<Config> {
-        FileConfig::load_from_path(path).map(|config| config.into(path))
+    pub fn load_from_path(path: &str) -> anyhow::Result<Config> {
+        let mut config = FileConfig::load_from_path(path).map(|config| config.into(path))?;
```

Comment:
We should add more context here to say that we are not able to load file from path using the `with_context` method

Reply:
Sure thing!